src/components/Amalgamation/BusinessStatus.vue

```diff
@@ -80,8 +80,8 @@ export default class BusinessStatus extends Vue {
         return 'This business is under limited restoration. It cannot be part of an amalgamation ' +
           'unless it is fully restored.'
 
-      // case AmlStatuses.ERROR_NEED_BC_COMPANY:
-      //   return 'You must add at least one BC company.'
+      case AmlStatuses.ERROR_NEED_BC_COMPANY:
```

Comment:
Spoke to Mihai about this. We indeed need as it is in the legacy system:
![MicrosoftTeams-image (1)](https://github.com/bcgov/business-create-ui/assets/122301442/383b4263-7a45-44bd-aa2c-44e9c78c354b)

I've tested it, it works:
![need one bc company](https://github.com/bcgov/business-create-ui/assets/122301442/cdff9561-bc25-4cff-b401-d9acb0f2caa3)

Please note the changes too in the other files.